vm/actor_interface/Cargo.toml

```diff
@@ -20,7 +21,8 @@ forest_bitfield = "0.1"
 num-bigint = { version = "0.1.1", package = "forest_bigint", features = ["json"] }
 forest_hash_utils = "0.1"
 forest_json_utils = "0.1.1"
-
+ipld_hamt = { path = "../../ipld/hamt" }
+ipld_amt = { path = "../../ipld/amt", features = ["go-interop"]}
```

Comment:
these should be defined in actors, like the other versions. This will just add confusion and unnecessary duplication of this dependency reference.

I see the TODO, but what's stopping this from being done now?

Reply:
Because there will be like a bazillion something errors because the interface has changed. Which means ill have to update the actors code to the new interface only to need to update it again when i actually update the actors.

Reply:
Oh I see, sure whatever works better for you then. This was opinionated I guess